app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -237,6 +245,42 @@ class DeveloperOptionsActivityTest {
     }
   }
 
+  @Test
+  fun testDeveloperOptionsFragment_clickMarkChaptersCompleted_opensMarkChaptersCompletedActivity() {
```

Comment:
can we have `configChange` variant of these 3 new test cases.

Reply:
Tests added.